### PR TITLE
Typing errors

### PR DESCRIPTION
**Pull request by Malleek**
Correction of various typing errors.

### CODE REVIEW

File: src/main/java/pipe/gui/GuiFrame.java [hunk 1]
```diff
@@ -232,17 +232,17 @@ public void actionPerformed(ActionEvent arg0) {
             currentTab.ifPresent(TabActions::verifySelectedQuery);
         }
     };
-    private final GuiAction workflowDialogAction = new GuiAction("Workflow analysis", "Analyse net as a TAWFN", KeyStroke.getKeyStroke(KeyEvent.VK_F, shortcutkey)) {
+    private final GuiAction workflowDialogAction = new GuiAction("Workflow analysis", "Analyses net as a TAWFN", KeyStroke.getKeyStroke(KeyEvent.VK_F, shortcutkey)) {
```
Comment (srba) [inline src/main/java/pipe/gui/GuiFrame.java:235]: There should be Analyse, not Analyses

File: src/main/java/pipe/gui/GuiFrame.java [hunk 2]
```diff
@@ -232,17 +232,17 @@ public void actionPerformed(ActionEvent arg0) {
...
         public void actionPerformed(ActionEvent e) {
             currentTab.ifPresent(TabActions::workflowAnalyse);
         }
     };
-    private final GuiAction smartDrawAction = new GuiAction("Automatic net layout", "Rearrange the Petri net objects", KeyStroke.getKeyStroke('D', KeyEvent.SHIFT_DOWN_MASK)) {
+    private final GuiAction smartDrawAction = new GuiAction("Automatic net layout", "Rearranges the Petri net objects", KeyStroke.getKeyStroke('D', KeyEvent.SHIFT_DOWN_MASK)) {
```
Comment (srba) [inline src/main/java/pipe/gui/GuiFrame.java:240]: Keep rearrange, not rearranges

File: src/main/java/pipe/gui/GuiFrame.java [hunk 3]
```diff
@@ -232,17 +232,17 @@ public void actionPerformed(ActionEvent arg0) {
...
         public void actionPerformed(ActionEvent e) {
             SmartDrawDialog.showSmartDrawDialog();
         }
     };
-    private final GuiAction mergeComponentsDialogAction = new GuiAction("Merge net components", "Open a composed net in a new tab and use approximated net if enabled", KeyStroke.getKeyStroke(KeyEvent.VK_C, (shortcutkey + InputEvent.SHIFT_MASK))) {
+    private final GuiAction mergeComponentsDialogAction = new GuiAction("Merge net components", "Opens a composed net in a new tab and use approximated net if enabled", KeyStroke.getKeyStroke(KeyEvent.VK_C, (shortcutkey + InputEvent.SHIFT_MASK))) {
```
Comment (srba) [inline src/main/java/pipe/gui/GuiFrame.java:245]: Keep Open

File: src/main/resources/Example nets/referendum-colored.tapn [hunk 4]
```diff
@@ -198,7 +198,7 @@
       </declarations>
     </structure>
   </declaration>
-  <query active="true" algorithmOption="CERTAIN_ZERO" approximationDenominator="0" capacity="12" colorFixpoint="false" discreteInclusion="false" enableOverApproximation="false" enableUnderApproximation="false" extrapolationOption="AUTOMATIC" gcd="false" hashTableSize="MB_16" inclusionPlaces="*NONE*" name="Six voted no" overApproximation="false" pTrie="false" partitioning="false" reduction="true" reductionOption="VerifyPN" searchOption="RANDOMHEURISTIC" symmetricVars="false" symmetry="false" timeDarts="false" traceOption="NONE" type="CTL" useQueryReduction="true" useSiphonTrapAnalysis="false" useStubbornReduction="true" useTarOption="false">
```
Comment (srba) [inline src/main/resources/Example nets/referendum-colored.tapn:201]: Why are the capacities modified?

File: src/main/resources/Example nets/referendum-colored.tapn [hunk 5]
```diff
@@ -198,7 +198,7 @@
...
+  <query active="true" algorithmOption="CERTAIN_ZERO" approximationDenominator="0" capacity="9" colorFixpoint="false" discreteInclusion="false" enableOverApproximation="false" enableUnderApproximation="false" extrapolationOption="AUTOMATIC" gcd="false" hashTableSize="MB_16" inclusionPlaces="*NONE*" name="Six voted no" overApproximation="false" pTrie="false" partitioning="false" reduction="true" reductionOption="VerifyPN" searchOption="RANDOMHEURISTIC" symmetricVars="false" symmetry="false" timeDarts="false" traceOption="NONE" type="CTL" useQueryReduction="true" useSiphonTrapAnalysis="false" useStubbornReduction="true" useTarOption="false">
```
Comment (srba) [inline src/main/resources/Example nets/referendum-colored.tapn:201]: Again modified capacity?

File: src/main/resources/Example nets/referendum-colored.tapn [hunk 6]
```diff
@@ -223,7 +223,7 @@
           
     </formula>
   </query>
-  <query active="true" algorithmOption="CERTAIN_ZERO" approximationDenominator="0" capacity="4" colorFixpoint="true" discreteInclusion="false" enableOverApproximation="false" enableUnderApproximation="false" extrapolationOption="null" gcd="false" hashTableSize="null" inclusionPlaces="*NONE*" name="Tie vote" overApproximation="false" pTrie="false" partitioning="true" reduction="true" reductionOption="VerifyPN" searchOption="RANDOMHEURISTIC" symmetricVars="true" symmetry="false" timeDarts="false" traceOption="NONE" type="CTL" useQueryReduction="true" useSiphonTrapAnalysis="false" useStubbornReduction="true" useTarOption="false">
```
Comment (srba) [inline src/main/resources/Example nets/referendum-colored.tapn:226]: Modified capacity?

File: src/main/resources/Example nets/referendum-timed-colored.tapn [hunk 7]
```diff
@@ -216,8 +216,8 @@
       </declarations>
     </structure>
   </declaration>
-  <query active="true" approximationDenominator="2" capacity="4" colorFixpoint="false" discreteInclusion="false" enableOverApproximation="false" enableUnderApproximation="false" extrapolationOption="AUTOMATIC" gcd="false" hashTableSize="MB_16" inclusionPlaces="*NONE*" name="Six voted no" overApproximation="false" pTrie="false" partitioning="false" query="EF Referendum_timed_colored.voted_no = 6" reduction="true" reductionOption="VerifyTAPNdiscreteVerification" searchOption="HEURISTIC" symmetricVars="false" symmetry="false" timeDarts="false" traceOption="NONE" useStubbornReduction="true" useTarOption="false"/>
```
Comment (srba) [inline src/main/resources/Example nets/referendum-timed-colored.tapn:219]: Modified capacities

**Review by srba — CHANGES_REQUESTED** [review]
Some small comments are added. Also, please, add your name to the "About" menu as the contributer to TAPAAL (We use alphabetical order here).

File: src/main/resources/Example nets/referendum-colored.tapn (hunk 4 above)
Comment (Malleek) [reply src/main/resources/Example nets/referendum-colored.tapn:201]: The 4 queries of referendum-colored and referendum-timed-colored are not satisfiable unless you put at least 9 extra tokens. There 10 voters in both case, so k-bound should be 10.

File: src/main/java/pipe/gui/GuiFrame.java (hunk 1 above)
Comment (Malleek) [reply src/main/java/pipe/gui/GuiFrame.java:235]: OK. I wanted to be consistent with the other tooltips, but I see why I shouldn't

**Comment by Malleek** [comment]
I made the changes, it should be good now

**Review by srba — APPROVED** [review]
Looks good

**Pull request merged**